src/game/objects/creatures/natla.c

```diff
@@ -82,6 +82,7 @@ void Natla_Control(int16_t item_num)
     int16_t tilt = 0;
     int16_t gun = natla->head_rotation * 7 / 8;
     int16_t timer = natla->flags & NATLA_TIMER;
+    int16_t facing = (int16_t)(size_t)item->priv;
```

Comment:
Do we need this double cast, here and in other places?

Reply:
Without it, the following compiler warning is generated. I think if it were an `int32_t` we would be OK, but as it's an angular property I believe the range needs to be limited to `int16_t`.

```
[2/9] Compiling C object Tomb1Main.exe.p/src_game_objects_creatures_natla.c.obj
../src/game/objects/creatures/natla.c: In function ‘Natla_Control’:
../src/game/objects/creatures/natla.c:85:22: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
   85 |     int16_t facing = (int16_t)item->priv;
```